packages/next-app/src/app/daoadmin/action/[actionId]/(components)/ActionAttributeContent.tsx

```diff
@@ -76,12 +84,25 @@ export function ActionAttributeContent() {
               <div className="flex flex-col gap-4">
                 {selectedAction?.fields.map((field) => {
                   return (
-                    <Input
-                      key={field.name}
-                      label={field.name}
-                      placeholder={field.placeholder}
-                      {...register(toSlug(`${field.name}`))}
-                    />
+                    <div key={field.name}>
+                      <Input
+                        label={field.name}
+                        placeholder={field.placeholder}
+                        {...register(toCamelCase(`${field.name}`))}
```

Comment:
also think this is very likely to lead to errors. You're not enforcing any interfaces with TS and even so, Typescript wouldn't know if a string is or not camelCase. If you're struggling with using the same string as a name to be displayed to the end user you can just add a separate `key` property, or add a new `label` property to the selectedAction fields so that you can use it as a display name and the key as an identifier in the form.